### ISSUE TITLE
[Bug] Media play buttons in tabs tray are gone for some sites

### ISSUE THREAD

**Issue by liuche** [mozilla-mobile/fenix#16109]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have the exact bug and steps to reproduce described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

## Steps to reproduce
Go to site https://www.collectivecreamery.com/podcast and click play

or cafe noise site https://mynoise.net/NoiseMachines/cafeRestaurantNoiseGenerator.php

Still works on Youtube.

### Expected behavior
Media icons are showing on tabs, persistent media notification in system notification tray.

### Actual behavior
Sound is playing, no notifications

![photo4938661080264190144](https://user-images.githubusercontent.com/1082253/96806289-89219b00-13c8-11eb-93d6-b64e8c0dcb7b.jpg)

### Device information

* Android device: Pixel 3
* Fenix version: Nightly 10/21

**Comment by mcarare** [comment]
I think this is related to many other issues regarding Media (https://github.com/mozilla-mobile/fenix/issues/7370, https://github.com/mozilla-mobile/fenix/issues/10740, https://github.com/mozilla-mobile/fenix/issues/4793, https://github.com/mozilla-mobile/fenix/issues/5290) and it is still waiting on integrating the GV API into AC: https://github.com/mozilla-mobile/fenix/issues/5290#issuecomment-699902435

This should be tested along with the other issues, when the new API is implemented.

**Comment by rocketsroger** [comment]
With the new MediaSession API, this should be fixed on nightly.  I tested with https://www.collectivecreamery.com/podcast and the media play button is showing.

However, for https://mynoise.net/NoiseMachines/cafeRestaurantNoiseGenerator.php, I tried on Chrome as well and there's no notification showing for the media.  I think in the cafe case it is a site issue.  We behave the same way as Chrome.
